rdrf/rdrf/management/commands/check_structure.py

```diff
@@ -0,0 +1,105 @@
+import sys
+from django.core.management import BaseCommand
+from rdrf.models import Registry
+from rdrf.models import Modjgo
+import yaml
+import jsonschema
+import errno
+import os
+
+explanation = "This command checks for schema validation errors"
+
+SCHEMA_LOCATIONS = ["/app/rdrf/rdrf/schemas/modjgo.yaml",
+                    "/env/src/django-rdrf/rdrf/rdrf/schemas/modjgo.yaml"]
+
```

Comment:
I don't like these docker container specific hardcoded paths here.
Can we do something about them?
For example get path based on a module's `__file__` or similar.